GNUmakefile

```diff
@@ -133,20 +134,17 @@ ifdef SKIP_DOCKER_BUILD
 ENVOY_INTEG_DEPS=noop
 endif
 
-# all builds binaries for all targets
-all: bin
+all: dev-build
 
 # used to make integration dependencies conditional
 noop: ;
 
-bin: tools
```

Comment:
@dnephin is `tools` no longer required?

Reply:
I believe that's correct. We shouldn't require any additional tools to build the binary. `tools` installs stuff for updating protos , re-generating the bindata (which you are removing), run lint, or generate mocks, but nothing that should be required to build the binary.